src/ggml-cpu/vec.h

```diff
@@ -0,0 +1,806 @@
+// Vectorized functions for fundamental operations
+
+#pragma once
+
+#include "ggml-impl.h"
+#include "simd-mappings.h"
+#include "ggml.h"
+
+#include <assert.h>
```

Comment:
Should be moved to `vec.cpp` and changed to `#include <cassert>`.

Reply:
Done, thanks!